workshop/Lab0/README.md

```diff
@@ -0,0 +1,112 @@
+
+or otherwise targeting your IBM Cloud Kubernetes Service cluster.
+
+# Install Helm
+
+1. Download and install [the Helm CLI](https://github.com/kubernetes/helm#install).
+2. Run `helm init` to install the tiller pod into your kube cluster. This pod is Helm's entry point
+for creating additional infrastructure inside your cluster.
+3. To allow the tiller pod permissions to perform changes inside your cluster, we need to define the
+Role Based Access Control (RBAC) policy for it by defining a new cluster role and giving it the
+appropriate permissions. Do this by running:
+
+```
+$ kubectl create clusterrolebinding tiller-cluster-admin \
+--clusterrole=cluster-admin \
+--serviceaccount=kube-system:default
+```
+
+# Install Service Catalog
+
+Service Catalog is installed via a Helm chart, a manifest that describes what to install and how to run it.
+
+1. First, add the Service Catalog repo to your helm by running 
+```
+$ helm repo add svc-cat https://svc-catalog-charts.storage.googleapis.com
+```
+You should see the repository listed in the output of `helm search service-catalog`.
+
+2. Install Service Catalog by running 
+```
+$ helm install svc-cat/catalog --name catalog --namespace catalog
+```
+
+You should be able to see the helm tiller and service catalog pods like so:
+```console
+$ kubectl get pods --all-namespaces
+NAMESPACE     NAME                                                  READY     STATUS    RESTARTS   AGE
+catalog       catalog-catalog-apiserver-6bc6bb895d-f2xhp            2/2       Running   0          8m
+catalog       catalog-catalog-controller-manager-5c7475f7d4-fwlfg   1/1       Running   1          8m
+kube-system   kube-addon-manager-minikube                           1/1       Running   0          8m
+kube-system   kube-dns-54cccfbdf8-vpvmn                             3/3       Running   0          8m
+kube-system   kubernetes-dashboard-77d8b98585-sw5gc                 1/1       Running   0          8m
+kube-system   storage-provisioner                                   1/1       Running   0          8m
+kube-system   tiller-deploy-865dd6c794-88njn                        1/1       Running   0          9m
+```
+
+# Install svcat
+
+svcat is a command line tool for interacting with Service Catalog. While Service Catalog
+can be intereacted with solely with kubectl, svcat provides a nicer UX that is customized
+to interact with Service Catalog specific resource types. It can be run as a kubectl plugin,
+or as a standalone binary. Download the version that corresponds to your operating system:
+
+* [Linux](https://download.svcat.sh/cli/latest/linux/amd64/svcat)
+* [OS X](https://download.svcat.sh/cli/latest/darwin/amd64/svcat)
+* [Windows](https://download.svcat.sh/cli/latest/windows/amd64/svcat.exe)
+
+Make the binary executable:
+
+* (Linux/OS X) `chmod +x svcat`
+* (Windows) Make sure the filename ends in .exe
+
+Move the file somewhere on your PATH:
+
+* (Linux/OS X) `mv svcat /usr/local/bin/`
```

Comment:
This will only work if they're root - which I suspect most people will not be. Maybe just skip this since if they can't figure out how to run the exe then they're pretty out of it  :-)

Reply:
ditto for chmod +x - seem unnecessary since you didn't also tell them how to use 'curl'  :-)

Reply:
eh, I'm not so sure about that. You can click on the link and download the binary manually, you don't have to use curl. I could add `sudo` to the command. I'll revise this section to have go over the steps without attempting to literally spell out the commands.